audioipc/src/codec.rs

```diff
@@ -154,7 +154,7 @@ where
 
     fn encode(&mut self, item: Self::In, buf: &mut BytesMut) -> io::Result<()> {
         trace!("Attempting to encode");
-        let encoded_len = serialized_size(&item);
+        let encoded_len = serialized_size(&item).unwrap();
```

Comment:
What's the likelihood of this ever failing?

I've been bitten too many times by `unwrap()` instead of handling errors in `cubeb-rs` and this crate that I'm a little gun shy of having unwrap here but it's probably going to be OK in this instance.